openpype/hosts/maya/plugins/publish/extract_playblast.py

```diff
@@ -118,7 +118,7 @@ def process(self, instance):
 
         # Need to explicitly enable some viewport changes so the viewport is
         # refreshed ahead of playblasting.
-        panel = cmds.getPanel(withFocus=True)
+        panel = cmds.getPanel(withFocus=True) or ""
```

Comment:
Should we maybe [try `cmds.playblast(activeEditor=True)` instead](https://help.autodesk.com/cloudhelp/2022/ENU/Maya-Tech-Docs/CommandsPython/playblast.html#flagactiveEditor)? Not sure if more reliable but it's what built-in playblasting does to get the active panel to playblast. I've seen it work even if e.g. scriptEditor was the panel with focus.

Reply:
Nice, yeah. The whole getting of the correct panel is an issue everywhere in this extractor. Will dig into `activeEditor`.

Might have to be a separate PR/issue though since this is just a hotfix.

Reply:
This actually works much better. Will update the PR in a bit.